app/controllers/accept_account_invitations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AcceptAccountInvitationsController < ApplicationController
+  def show
+    not_found unless received_invitation &&
+                     invitation_belongs_to_current_user? &&
+                     not_accepted_invitation?
+  end
+
+  def update
+    received_invitation.update!(accepted_at: Time.now)
```

Comment:
```suggestion
    received_invitation.update!(accepted_at: Time.current)
```

Use current instead of now please as current takes time zone into account